open-metadata-implementation/user-interfaces/presentation-server/nodejs/src/client/components/TypeExplorer/README.md

```diff
@@ -0,0 +1,76 @@
+## Type Explorer User Interface
+
+The Type Explorer (Tex) interface provides a way to explore the Egeria type system. It can be used to retrieve the types supported by a chosen server. The type information is then displayed both textually (on the left side) and diagrammatically (on the right side) with a choice of diagrams. One of the diagrams shows the entity inheritance tree and the other diagram shows the types of relationships that an entity type supports.
+
+![Tex Interface](image1)
+
+### Deployment
+The Tex interface runs under the Presentation Server component of Egeria, and it uses the Type Explorer (Tex) View Service in order to retrieve the information you ask for. For details of the Presentation Server and View Server please refer to their readme information.
+
+
+### Configuration
+The Tex View Service is an Integration View Service and is configured using `IntegrationViewServiceConfig`. This contains the name and status of the View Service and also contains a list of the resources that the Tex interface is permitted to access. These resources are the platforms and servers that the user will be able access. The configured servers will appear in the selectors in the Tex interface. All requests to the Tex View Service REST API are based on these configured named resources. When a user selects a server name from the selector lists, the interface sends the resource name to the Tex View Service, which resolves the server name to a resource endpoint to identify the URL needed to send a request to the server. 
+
+In the Tex View Service configuration, the view service administrator configures a list of Resource Endpoints to represent platforms and servers. Every `ResourceEndpointConfig` needs a `resourceCategory`, which is set to either `"Platform"` or `"Server"`. A platform `ResourceEndpointConfig` must have a unique `platformName` and a `platformRootURL` and can have an optional `description` property. A server `ResourceEndpointConfig` for a server must have a `serverName` and the `platformName` of a configured platform resource endpoint. A server can have an optional `description` property. In an Egeria deployment, a server may be deployed to multiple platforms - this is typically used for clustering - and a server ResourceEndpointConfig contains a property called serverInstanceName which must be unique and is used to name a specific instance of a server. For example, you could configure a pair of server resource endpoints called "Server1@PlatformA" and "Server1@PlatformB". The serverInstanceName is used to display the resource in the user interface selector lists. 
+
+An example configuration with two platforms and two servers is shown below:
+
+```
+{
```

Comment:
In this example where are the resourse names?

Reply:
They are the serverInstanceName and platformName, depending on the category. 
I think I have disambiguated this with the changes made in response to your previous comment - I refer explicitly to (back-quoted) "platformName" or "serverName" now, rather than "resource name".

Have thoroughly beefed up the description just before the example config.